<ui version="4.0" >
 <class>MvdLinksPage</class>
 <widget class="QWidget" name="MvdLinksPage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Links</string>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="5" >
    <widget class="QLabel" name="infoLabel" >
     <property name="contextMenuPolicy" >
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="text" >
      <string>Use the context menu to open the page in a browser.
Toggle the checkbox to set a URL as default.</string>
     </property>
     <property name="margin" >
      <number>4</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5" >
    <widget class="MvdTreeWidget" name="linksWidget" >
     <column>
      <property name="text" >
       <string>1</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="imdbLabel" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>IMDb ID:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QLineEdit" name="imdbInput" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>60</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>60</width>
       <height>20</height>
      </size>
     </property>
     <property name="inputMask" >
      <string>9999999;#</string>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="cursorPosition" >
      <number>7</number>
     </property>
    </widget>
   </item>
   <item row="3" column="2" >
    <spacer>
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Minimum" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>16</width>
       <height>23</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="3" >
    <widget class="QPushButton" name="imdbButton" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Visit &amp;IMDb movie page</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4" >
    <spacer>
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>244</width>
       <height>23</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MvdTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>treewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
